be/src/io/fs/benchmark/s3_benchmark.hpp

```diff
@@ -52,4 +55,164 @@ class S3ReadBenchmark : public BaseBenchmark {
     size_t _bytes_read = 0;
 };
 
+class S3SizeBenchmark : public BaseBenchmark {
```

Comment:
This is meaningless. Because the method `size()` is just return the `_size` field.
No action to do with S3.

Reply:
You need to call `s3_file_system`'s `file_size_impl()`